### DISCUSSION TITLE
不知道算不算 Bug

### DISCUSSION THREAD

**Discussion by wuliaodexiaoluo** [Meloong-Git/PCL#4146] [提问] [answered]
![image](https://github.com/Hex-Dragon/PCL2/assets/160083863/b647f55a-bc73-4742-be08-9e03d9a76199)
测试 #4136  时出现的 Mod 重复项（也可能是 API 返回不一致没法去重？）
[Log1.txt](https://github.com/user-attachments/files/16070545/Log1.txt)

**Comment by allMagicNB** [comment]
刻意的设计。

**Comment by Deep-Dark-Forest** [comment, marked as answer]
一个来自于mr，一个来自于cf，根据#1318 这必须分开显示
